app/controllers/UsersController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Webhooks\Models\Subscription;
+use Webhooks\Models\Event;
+use Webhooks\Models\Url;
+use Webhooks\Models\User;
+
+class UsersController extends BaseController 
+{
+
+    public function subscription()
+    {
+        $userId = 1;
+        $subscriptions = User::find(1)->subscriptions;
+        foreach ($subscriptions as $subscription) {
+            $event = Event::where('id', $subscription->event_id)->first();
+            $url = Event::find($subscription->event_id)->urls()->where('subscription_id', $subscription->id)->get()->toArray();
+            $urls['id'] = array_fetch($url,'id');
+            $urls['callback_url'] = array_fetch($url,'callback_url');
+            $eventInformation[] = [
```

Comment:
subscriptions
lines 22,24 and 25 can be grouped under [event], since those 3 attributes represents an event object